packages/headless/src/features/facets/range-facets/date-facet-set/date-facet-set-slice.ts

```diff
@@ -78,6 +89,7 @@ function buildDateFacetRequest(
     currentValues: [],
     preventAutoSelect: false,
     type: 'dateRange',
+    rangeAlgorithm: 'equiprobable',
```

Comment:
Should the default be `even` to keep the same behaviour as right now?

Reply:
I see, this change would be considered breaking to existing implementations. Ideally we would change the default starting in the v1 release,